packages/manager/src/features/Linodes/MigrateLinode/ConfigureForm.tsx

```diff
@@ -119,18 +144,15 @@ export const ConfigureForm = React.memo((props: Props) => {
 
         <StyledMigrationBox>
           <RegionSelect
-            regions={
-              regions?.filter(
-                (eachRegion) => eachRegion.id !== currentRegion
-              ) ?? []
-            }
             textFieldProps={{
               helperText,
             }}
             currentCapability="Linodes"
             errorText={errorText}
+            geckoEnabled={linodeIsInEdgeRegion}
```

Comment:
I'm finding the `geckoEnabled` prop to be a bit confusing. Could we make this more descriptive of what it does? 

Currently, it looks like it enables filtering, but it looks like we are also filtering the regions in this component. 

```suggestion
            geckoEnabled={linodeIsInEdgeRegion}
```

Reply:
Good point, we're filtering twice in slightly different ways. Will look into unifying this

Reply:
@bnussman-akamai addressed in https://github.com/linode/manager/pull/10238/commits/cb032f0ad4a70cd96cbeacc3b7068f231e6cab00